src/Attribute/FutureReady.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Attribute;
+
+/**
+ * Marks nodes that are ready to accept a TwigFilter/TwigFunction/TwigTest instead of just a name.
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class FutureReady
```

Comment:
Not sure how to call it.

Reply:
I'd rather avoid `Future` in the name because we won't remove this attribute in 4.0.0 (to make it easy to migrate by supporting multiple versions) and in that version, it won't be about *future* anymore.
But I don't have a good idea for a name either.

Reply:
Maybe it's time to put a name on functions/filters/tags. See #4192 where I introduce the notion of Twig callables.